app/client/components/pages/settings/settingsPage.jsx

```diff
@@ -19,6 +20,7 @@ export default class settingsPage extends React.Component {
   render() {
     return (
       <WrapperDiv>
+        <DemoGenericRedux />
```

Comment:
Good for demo, but maybe not necessary once it's been approved?

Reply:
Yeah exactly my thoughts!